dotcom-rendering/src/web/components/Discussion.tsx

```diff
@@ -91,44 +132,83 @@ export const Discussion = ({
 		}
 	}, [hasCommentsHash]);
 
+	useEffect(() => {
+		const pendingElements = document?.querySelectorAll<HTMLElement>(
+			'.discussion > .pending',
+		);
+		pendingElements?.forEach((element) => {
+			element.classList.add('reveal');
+			element.classList.remove('pending');
+		});
+	}, []);
+
+	useEffect(() => {
+		// There's no point showing the view more button if there isn't much more to view
+		if (commentCount === 0 || commentCount === 1 || commentCount === 2) {
```

Comment:
Is this not simply checking that there are less than 3 comments?

```suggestion
		if (commentCount < 2) {
```

Reply:
But seriously, I prefer being plain and explicit. What if there's an edge case if `commentCount` is zero or something like that? I find the original code instantly understandable and readable, even if it's a bit verbose.

Reply:
Okay—just curious as I had not seen this pattern before.

I should have written `< 3` or `<= 2`, but I'm fine with the three conditions if intentional!